keyboards/yellowbrickroad/readme.md

```diff
@@ -0,0 +1,23 @@
+# yellowbrickroad
+
+![yellowbrickroad](https://github.com/3araht/yellowbrickroad/blob/main/pictures/YellowBrickRoad_Title.jpg)
+
+"There’s no place like home.” I designed this keyboard since I wanted to go back to Kansas, LOL.<br>
+yellowbrickroad is a MIDI keyboard which has 53 keys (= 4 octaves+ ) placed spirally. Raspberry Pi Pico is used as the "brain" of yellowbrickroad.<br>
+
+Size: 20 cm (approx 8 inches) x 23 cm (approx 9 inches).<br>
+It fits in an A4 / Letter size paper.<br>
+
+* Keyboard Maintainer: [3araht](https://github.com/yourusername)
+* Hardware Supported: yellowbrickroad keyboard, a MIDI keyboard.
+* Hardware Availability: [Yushakobo](https://shop.yushakobo.jp/collections/keyboard/products/3442), [BOOTH](https://3araht.booth.pm/). Click [here](https://www.tenso.com/en/static/lp_shop_booth) for BOOTH overseas shipping!
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

Reply:
This request has been applied.